tests/mocha/server/CollectionsTest.js

```diff
@@ -0,0 +1,29 @@
+MochaWeb.testOnly(function() {
+  describe('Collections', function() {
+
+    Gizmos = new Mongo.Collection("gizmos");
```

Comment:
It does, and I'll fix it. I just copied the pattern from our prod code, but that's not appropriate here.